### DISCUSSION TITLE
Adding Traditional Chinese README

### DISCUSSION THREAD

**Discussion by stonkol** [OpenTabletDriver/OpenTabletDriver#3150] [General]
### Description

I would like to translate the README file to Traditional Chinese. Would it be possible?

### Acknowledgements

- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
You are completely welcome to do that, see our [CONTRIBUTING.md](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/CONTRIBUTING.md).
